rest-lib-utils/pom.xml

```diff
@@ -166,6 +166,11 @@
 			<artifactId>commons-io</artifactId>
 			<version>2.7</version>
 		</dependency>
+		<dependency>
+			<groupId>com.sun.activation</groupId>
+			<artifactId>jakarta.activation</artifactId>
+			<version>2.0.1</version>
+		</dependency>
```

Comment:
Why is this new dependency necessary?

Reply:
When building with Java 11, one of the tests in rest-lib-utils failed due to a ClassNotFoundException (`com.sun.activation.registries.LogSupport.class`), and I guessed this is due to the [removal of activation framework in Java 11](https://docs.microsoft.com/en-us/azure/developer/java/fundamentals/transition-from-java-8-to-java-11). Adding this dependency resolved the issue, but now I cannot reproduce the error anymore (maybe due to the upgrade to Spring Boot 2.5).

Reply:
I confirm that it is not needed anymore, thank you for questioning its use.